ci/jenkins/testkomodo.sh

```diff
@@ -19,8 +19,10 @@ start_tests () {
 
 start_integration_test () {
 
-    echo "Downloading chromedriver ..."
-    wget -O chromedriver.zip https://chromedriver.storage.googleapis.com/108.0.5359.71/chromedriver_linux64.zip
+    chromium_version=`chromium-browser --version | grep -zoP '\d+\.\d+\.\d+'`
+    chromedriver_version=`curl -s https://chromedriver.storage.googleapis.com/LATEST_RELEASE_$chromium_version`
```

Comment:
Should we use dollar-parens form instead of backticks?  http://mywiki.wooledge.org/BashFAQ/082

Reply:
Doh, agreed.